tests/SpecTests/FunctionalTestCase.php

```diff
@@ -185,31 +185,55 @@ protected function dropTestAndOutcomeCollections()
     {
         $context = $this->getContext();
 
-        $collection = $context->getCollection();
-        $collection->drop($context->defaultWriteOptions);
+        if ($context->bucketName !== null) {
+            $bucket = $context->getGridFSBucket();
```

Comment:
`Bucket::drop()` currently does not support any options. This may change in [SPEC-921](http://jira.mongodb.org/browse/SPEC-921) when session support is introduced, but I don't expect it (or any other GridFS methods) will support a write concern option like the Database or Collection drop methods do.

We can consider passing `$context->defaultWriteOptions` to `getGridFSBucket()` here to ensure a majority write concern is used for the ensuing `drop()` operation.

I'll note that only the retryable writes and transactions spec test runners care about using a majority write concern, so we wouldn't need to define a majority write concern in `Context::fromRetryableReads()` as we do in the other spec methods.

Note: This relates to my earlier comment in `getGridFSBucket()`.

I'm OK with whatever you'd like to do here.